airflow/www/templates/airflow/trigger.html

```diff
@@ -27,11 +27,93 @@
+        <strong style="color: red"> *</strong>
+      {%- endif -%}
+    : </label>
+  </td>
+  <td>
+    {% if "custom_html_form" in form_details.schema %}
+      {{ form_details.schema.custom_html_form | replace("{name}", "element_" + form_key) | replace("{value}", form_details.value) }}
+    {% elif "type" in form_details.schema and form_details.schema.type == "boolean" %}
+      <label for="element_{{ form_key }}" control-label="">
+        <input class="switch-input" name="element_{{ form_key }}" id="element_{{ form_key }}" type="checkbox"
+          {%- if form_details.value %} checked="checked"{%- endif -%}/>
+        <span class="switch" aria-hidden="true"></span>
+      </label>
+    {% elif "format" in form_details.schema and "date-time" in form_details.schema.format %}
+      <div class="input-group datetime datetimepicker">
+        <span class="input-group-addon"><span class="material-icons cursor-hand">calendar_today</span></span>
+        <input class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" type="text" valuetype="date"
+          {%- if not "null" in form_details.schema.type %} required=""{%- endif %} value="{% if form_details.value %}{{ form_details.value }}{% endif %}" />
+      </div>
+    {% elif "format" in form_details.schema and "date" in form_details.schema.format %}
+      <div class="input-group datetime datepicker">
+        <span class="input-group-addon"><span class="material-icons cursor-hand">calendar_today</span></span>
+        <input class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" type="text" valuetype="date"
+          {%- if not "null" in form_details.schema.type %} required=""{%- endif %} value="{% if form_details.value %}{{ form_details.value }}{% endif %}" />
+      </div>
+    {% elif "format" in form_details.schema and "time" in form_details.schema.format %}
+      <div class="input-group datetime timepicker">
+        <span class="input-group-addon"><span class="material-icons cursor-hand">calendar_today</span></span>
+        <input class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" type="text" valuetype="date"
+          {%- if not "null" in form_details.schema.type %} required=""{%- endif %}
+          value="{% if form_details.value %}{{ form_details.value[0:2] }}00-01-01 {{ form_details.value }}{% endif %}" />
+      </div>
+    {% elif "enum" in form_details.schema and form_details.schema.enum %}
+      <select class="my_select2 form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" data-placeholder="Select Value"
+        value="{% if form_details.value %}{{ form_details.value }}{% endif %}" onchange="updateJSONconf();"
+        {%- if not "null" in form_details.schema.type %} required=""{% endif %}>
+        {% for option in form_details.schema.enum -%}
+        <option>{{ option }}</option>
+        {% endfor -%}
+      </select>
```

Comment:
These lines generate a drop-down select:
Shot 1 - view:
![image](https://user-images.githubusercontent.com/95105677/214950812-cd440798-62a2-43c3-a30f-33c10762f601.png)

Shot 2 - dowp down open incl. filter - same linke on other select boxes in UI:
![image](https://user-images.githubusercontent.com/95105677/214950761-94d22897-09f0-48f0-a1f2-7af7f4fb7d24.png)